composer.json

```diff
@@ -89,7 +89,7 @@
         "willdurand/hateoas-bundle": "^2.0"
     },
     "conflict": {
-        "doctrine/doctrine-bundle": "<2.0 || ^3.0",
```

Comment:
to discuss with other, cause it's the same behaviour than having "doctrine/doctrine-bundle "^2.0" in normal requirements.